index.html

```diff
@@ -0,0 +1,48 @@

```

Comment:
please make sure to use `<h1>` instead of `<h2> `tag it is best practice to always use semantics like one `<h1>` tag per page. (because it is good for SEO and accessibility).